.github/workflows/gmc-e2e.yaml

```diff
@@ -56,11 +56,14 @@ jobs:
           echo "KUBECTL_TIMEOUT_SECONDS=60s" >> $GITHUB_ENV
           echo "continue_test=true" >> $GITHUB_ENV
           echo "should_cleanup=false" >> $GITHUB_ENV
-          echo "skip_validate=true" >> $GITHUB_ENV
           echo "APP_NAMESPACE=$APP_NAMESPACE"
 
-      - name: Kubectl install
-        id: install
+      - name: Run tests
+        id: run-test
+        env:
+          HUGGINGFACEHUB_API_TOKEN: ${{ secrets.HUGGINGFACEHUB_API_TOKEN }}
```

Comment:
Then the action have to be `pull_request_target` or the forked PR not able to test.

Reply:
Yes. I will change it. Current version is just for testing.